credoai/integration.py

```diff
@@ -71,21 +67,21 @@ class Metric(Record):
     def __init__(self,
             metric_type,
             value,
-            dataset_label,
+            dataset_variant = None,
             **metadata):
         super().__init__('metrics', **metadata)
         self.metric_type = metric_type
         self.value = value
-        self.dataset_label = dataset_label
+        self.dataset_variant = dataset_variant
         self.config_hash = dict_hash({k:v for k,v in self.__dict__.items() 
                                       if k!='value'})
     
     def _struct(self):
         return {
-            'metric_id': self.config_hash,
+            'key': self.config_hash,
             'type': self.metric_type,
             'value': self.value,
-            'dataset': self.dataset_label,
+            'dataset_variant_id': self.dataset_variant,
             'metadata': {'creation_time': self.creation_time,
                         **self.metadata},
             '$type': 'model_metrics'
```

Comment:
Please add `updated_at` attribute as well. Sever will use this time as a metric updated time.(It will be implemented soon)
```
'updated_at': self.creation_time.
```